app/models/goods_nomenclature.rb

```diff
@@ -190,19 +190,15 @@ def chapter_id
   end
 
   def heading_code
-    heading_short_code + '0' * 6 unless chapter?
```

Comment:
is the check redundant?

Reply:
There are circumstances when the short code aren't present so I did need to check it, yeah. Basically when the current goods nomenclature is a chapter - but I've moved the decision making to the view